### PR TITLE
Restyled login page and made the header on side drawer larger

### PR DESCRIPTION
**Pull request by maycie-morris**
The login page was completely restyled, I removed just the login button which redirected to the Okta sign in widget and instead had the widget display on the login page itself. I also edited the SideDrawer.js and made the "Human Rights First" header very slightly bigger.

### CODE REVIEW

File: src/components/pages/Login/LoginContainer.js [hunk 1]
```diff
@@ -56,65 +33,25 @@ const LoginContainer = () => {
...
-      </LoginButtonDiv>
-
-      {login && <div id="sign-in-widget" />}
-      {login && <CloseLogin onClick={toggleLogin}>X</CloseLogin>}
-      {login && <OverlayDiv></OverlayDiv>}
-    </LoginDiv>
+    <StyledLogin>
+      <div class="background-image" aria-label="cosmetic background image" />
+      <div id="sign-in-widget" aria-label="login form" />
+    </StyledLogin>
```
Comment (aelise17264) [inline src/components/pages/Login/LoginContainer.js:54]: much cleaner, easier for user to login

Comment (aelise17264) [inline src/components/pages/Login/LoginContainer.js:54]: Much cleaner, easier for the user to find the login form & sign in to their account, style closer to that of the human rights asylum website, not cluttered up by words and logos

**Review by aelise17264 — APPROVED** [review]
Much cleaner and professional looking landing page for login

**Review by tzonglian — APPROVED** [review]
Great job!

**Pull request merged**